src/main/webapp/app/course/dashboards/assessment-dashboard/assessment-dashboard-information.component.html

```diff
@@ -0,0 +1,90 @@
+<div class="row mt-3">
+    <div class="col-md-8">
+        <div class="card">
+            <div class="card-header bg-primary text-white" style="font-size: 1.3rem">
+                <h4 class="card-title">{{ 'artemisApp.assessmentDashboard.informationAboutYourAssessments' | translate }}</h4>
+            </div>
+            <div class="card-body" style="font-size: 1.1rem">
+                <!-- Information about assessments for course mode -->
+                <ul *ngIf="!isExamMode">
+                    <!-- Your assessments: -->
+                    <li>
+                        <b jhiTranslate="">{{ 'artemisApp.assessmentDashboard.yourAssessments' | translate }}:</b> {{ numberOfTutorAssessments }}
```

Comment:
This is probably causing the key is null error.

Reply:
yes probably, I removed it.